woocommerce-gateway-ebanx/services/class-wc-ebanx-one-click.php

```diff
@@ -428,20 +292,14 @@ protected function customer_has_ebanx_required_data() {
 		$_POST['billing_city']      = $this->get_user_billing_address()['city'];
```

Comment:
We're not using superglobals anymore, please replace `$_POST[<key>] = <value>` with `empty(WC_EBANX_Request::set(<key>, <value>))`